Makefile

```
@@ -1,13 +1,31 @@
+
+dev-env: set-git set-pre-commit set-test set-pipreqs
+dep: extract-requirements
+test: pytest
+clean: clean-pyc clean-test
+
+##### dev-env #####
 set-git:
 	git config --local commit.template .gitmessage.txt
 
-set-test:
-	pip install --no-cache-dir pre-commit pylint pytest pytest-cov
+set-pre-commit:
+	pip3 install --no-cache-dir pre-commit==2.11.1
 	pre-commit install
 
-test:
-	pre-commit run -a;make clean
+set-test:
+	pip3 install --no-cache-dir pytest==6.2.1 pytest-cov==2.10.1 pytest_xdist==2.2.0
+
+set-pipreqs:
+	pip3 install --no-cache-dir pipreqs==0.4.10
+
```

```suggestion
set-all:
    make set-git
    make set-pre-commit
    make set-tests
    make sete-pipreqs
```
를 추가하는게 어떨까요?

@pretraining/authors

---

저는 좋습니다. `set-all`을 해도 되구요

Makefile 상단에 `all: set-git set-pre-commit set-tests set-pipreqs` 을 기술해두면,
그냥 make하나도 모두 동작하게 됩니다.

---

@ssaru 네 해주세요!

---

이 부분에 대해서 더 고민해보았는데요. `set-all`은 `dev-env`와 같은 내용인 것 같습니다.

`dev-env`로 했을 때, 명령어는 `make dev-env`가 되어서, 개발 환경을 셋팅하는구나라고 이해할 수 있을 것 같은데요,,
`set-all`로 하면, `all`이라는 단어가 어떤 의미를 주는지 애매할 것 같습니다.
실제로 `set-all`을 한다고해서, `Makefile`에 기술된 모든 동작을 셋팅하진 않는 것 같구요.

---

`pipreqs`는 오타가 있는데 그부분 제외하면 모두 동작하지 않나요?

---

`all`에 반영하였습니다!